fdbserver/SimEncryptKmsProxy.actor.h

```diff
@@ -36,7 +36,7 @@
 #include "flow/actorcompiler.h" // This must be the last #include.
 
 using SimEncryptKeyId = uint64_t;
```

Comment:
I updated the usage elsewhere to unify the types to the one defined in EncryptUtils.h for the similar reason (as system key-space and auth-header domain addition was done later); can you please update the file to use the defines from EncryptUtils.h?
Thanks.